crates/shaders/src/compile/preprocess.rs

```diff
@@ -153,6 +166,14 @@ pub fn preprocess(
             if line.starts_with("let ") {
                 output.push_str("const");
                 output.push_str(&line[3..]);
+            } else if let Some(idx) = line.find("var<storage>") {
+                if cfg!(feature = "force_rw_storage") {
+                    let mut line = line.to_string();
+                    line.replace_range(idx..12, "var<storage, read_write>");
```

Comment:
This should either be `idx..idx+12` or we should use `starts_with` rather than `find`.

Reply:
Ah good catch. I was getting lucky because the shaders keep the binding index declarations in a separate line. I'll fix this.